BannerlordTwitch/BLTAdoptAHero/Actions/AdoptAHero.cs

```diff
@@ -260,6 +294,19 @@ private static (bool success, string message) ExecuteInternal(string args, strin
                 CharacterDevelopmentCampaignBehaivor.DevelopCharacterStats(newHero);
             }
 
+            if (settings.JoinPlayerClan)
+            {
+                if (Clan.PlayerClan.CompanionLimit < Clan.PlayerClan.Companions.Count)
+                {
+                    newHero.Clan = Clan.PlayerClan;
+                }
+            }
+
+            if (settings.JoinPlayerCompanion)
```

Comment:
Its possible to add as companion but NOT add to party I guess? Seems a bit weird, maybe `JoinPlayerCompanion` should imply join party? Otherwise they are at some random town still. But maybe that is good? I don't know...

Reply:
I think it's a right idea to avoid option that will do some weird comportment happen with some configuration. In this way, you are right. Why don't setup a suboption for JoinPlayerCompanion which will be "in party". This way, i don't think there is weird comportment.